### PR TITLE
mh_branch

### PR DESCRIPTION
**Pull request by akenzua**
[TP12275](https://maps.tpondemand.com/restui/board.aspx#page=board/5175634625349962976&appConfig=eyJhY2lkIjoiOUM2MjFGQzAzOEIzQTVDNDU1NjRCMUIxQ0FENkZBOTYifQ==&boardPopup=userstory/12275/silent)

This PR create a version 2 of the library for Money Helper. It uses a different font.

### CODE REVIEW

**Comment by davidtrussler** [comment]
Can you remove the previous commit `8c8de55d33a2dff958c4ac8244ea7f3e74f7dadd` from the master branch before we review this one please? Or alternatively create a PR to revert it.

**Comment by davidtrussler** [comment]
Can we remove `stormy-eyrie-80273` (whatever that is 🤔 ) from this PR? It doesn't form part of this work and doesn't need to be added as part of it. 😄

**Comment by akenzua** [comment]
> Can we remove `stormy-eyrie-80273` (whatever that is 🤔 ) from this PR? It doesn't form part of this work and doesn't need to be added as part of it. 😄

I am not sure of what it does but from the naming, it looks like its used by heroku. I didnt specifically touch it. Left to me, I will say we leave it.

**Comment by davidtrussler** [comment]
> > Can we remove `stormy-eyrie-80273` (whatever that is 🤔 ) from this PR? It doesn't form part of this work and doesn't need to be added as part of it. 😄
> 
> I am not sure of what it does but from the naming, it looks like its used by heroku. I didnt specifically touch it. Left to me, I will say we leave it.

Well it won't break anything I guess but it clutters up the codebase is all. It can be easily removed with `git rm stormy-eyrie-80273` so I would prefer to do that. Not a game-changer though.

**Comment by akenzua** [comment]
Removed

**Comment by davidtrussler** [comment]
We still need to actually update the version and publish the library so that it can be used on the project. The version is in `package.json` and publishing instructions are in the README. I have never done this but in this PR I would expect to see a change to the version number at the very least.

**Comment by akenzua** [comment]
Yeah. Thats true. Its changed.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@moneypensionservice/directories",
-  "version": "1.9.3",
+  "version": "2.0",
```
Comment (davidtrussler) [inline package.json:3]: We should use the `major.minor.patch` syntax here, i.e. `2.0.0`

**Review by davidtrussler — APPROVED** [review]
Nice work! LGTM! 👍

**Pull request merged**